src/index.ts

```diff
@@ -0,0 +1,89 @@
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { StdioServerTransport } from "@modelcontextprotocol/sdk/server/stdio.js";
+import { z } from "zod";
+import { getChainSpec, getRollupCodesName, getRollupMarkdownFields, listRollupsFromDocs } from './helpers.js'
+
+// Create server instance
+const server = new McpServer({
+  name: "rollup-codes",
+  version: "1.0.0",
+  capabilities: {
+    resources: {},
+    tools: {},
+  },
+});
+
+server.tool(
+  "getRollupSpecs",
+  "Returns structured JSON data for a given rollup, including metadata, opcodes, precompiles, system contracts, and more. It supports: Abstract, Arbitrum, Base, Blast, Ink, Kakarot, Linea, Optimism, Polygon ZKevm, Scroll, Soneium, Taiko, World Chain, Zircuit and ZKsync Era",
```

Comment:
Let's figure out a way to list the supported rollups by loading the data from the submodules instead of hardcoding it. if we add new rollup we want for the MCP server to automatically be able to serve rollup data without requiring the users to update their servers